specification/vmware/Microsoft.AVS/routes.tsp

```diff
@@ -0,0 +1,946 @@
+    }
+  )
+  @armResourceUpdate(PlacementPolicy)
+  @patch
+  update(
+    ...ResourceInstanceParameters<PlacementPolicy>,
+
+    @body
+    @doc("The placement policy properties to be updated.")
+    placementPolicyUpdate: PlacementPolicyUpdate,
+  ): ArmResponse<PlacementPolicy> | {
+    ...AcceptedResponse;
+    ...Azure.Core.Foundations.RetryAfterHeader;
+    ...LocationHeader;
+
+    @body
+    @doc("The updatd placement policy.")
+    placementPolicy: PlacementPolicy;
+  } | ErrorResponse;
+
+  delete is ArmResourceDeleteAsync<PlacementPolicy>;
+}
+
+@@projectedName(PlacementPolicies.createOrUpdate::parameters.resource,
+  "json",
+  "placementPolicy"
+);
+@@OpenAPI.extension(PlacementPolicies.createOrUpdate::parameters.resource,
+  "x-ms-client-name",
+  "placementPolicy"
+);
+
+@armResourceOperations
+interface ScriptPackages {
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  @operationId("ScriptPackages_List")
+  list is ArmResourceListByParent<ScriptPackage>;
+
+  get is ArmResourceRead<ScriptPackage>;
+}
+
+@armResourceOperations
+interface ScriptCmdlets {
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  @operationId("ScriptCmdlets_List")
+  list is ArmResourceListByParent<ScriptCmdlet>;
+
+  get is ArmResourceRead<ScriptCmdlet>;
+}
+
+@armResourceOperations
+interface ScriptExecutions {
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "Can not change existing operationId."
+  @operationId("ScriptExecutions_List")
+  list is ArmResourceListByParent<ScriptExecution>;
+
+  get is ArmResourceRead<ScriptExecution>;
+
+  createOrUpdate is ArmResourceCreateOrUpdateAsync<ScriptExecution>;
+
+  delete is ArmResourceDeleteAsync<ScriptExecution>;
+
+  @doc("Return the logs for a script execution resource")
+  @autoRoute
+  @armResourceAction(ScriptExecution)
+  @post
+  getExecutionLogs(
+    ...ResourceInstanceParameters<ScriptExecution>,
+
+    #suppress "@azure-tools/typespec-azure-core/request-body-problem"
+    @body
+    @doc("Name of the desired output stream to return. If not provided, will return all. An empty array will return nothing.")
+    scriptOutputStreamType?: ScriptOutputStreamType[],
+  ): ArmResponse<ScriptExecution> | ErrorResponse;
+}
+
+@@projectedName(ScriptExecutions.createOrUpdate::parameters.resource,
+  "json",
+  "scriptExecution"
+);
+@@OpenAPI.extension(ScriptExecutions.createOrUpdate::parameters.resource,
+  "x-ms-client-name",
+  "scriptExecution"
+);
+
+@doc("The location envelope.")
+model LocationHeader {
+  @doc("The Location header contains the URL where the status of the long running operation can be checked.")
+  @header("Location")
+  location?: string;
+}
+
+@armResourceOperations
+interface IscsiPaths {
+  list is ArmResourceListByParent<IscsiPath>;
+
+  get is ArmResourceRead<IscsiPath>;
+
+  createOrUpdate is ArmResourceCreateOrUpdateAsync<IscsiPath>;
```

Comment:
>IscsiPath [](http://example.com/codeflow?start=51&length=9)

it doesn't seem to have any writable properties? is it creatable anyway?

Reply:
`networkBlock` has `create`